pkg/settings/settings.go

```diff
@@ -78,6 +80,7 @@ const (
 	HarvesterCSICCMSettingName                        = "harvester-csi-ccm-versions"
 	StorageNetworkName                                = "storage-network"
 	DefaultVMTerminationGracePeriodSecondsSettingName = "default-vm-termination-grace-period-seconds"
+	SupportBundleExistTimeLimitSecondsSettingName     = "support-bundle-exist-time-seconds"
```

Comment:
How about `support-bundle-gc-timeout`? Thanks @ibrokethecloud

Reply:
How about `support-bundle-expired-time` ? `gc-timeout` sounds like it encounters timeout while gcing.

Reply:
@bk201 @ibrokethecloud How about `support-bundle-expired-time` ? `gc-timeout` sounds like it encounters timeout while gcing.

Reply:
`support-bundle-expiration-time` ?

Reply:
I think `support-bundle-expiration-time` would be nice